docs/RegistrationChecklist.md

```diff
@@ -0,0 +1,58 @@
+# Minimum Ontology Registration Request Checklist
+
+The _Minimum Ontology Registration Request Checklist_ (MORRC) is intended to provide a first pass to fairly evaluate ontology registration requests to the OBO foundry. 
+The goal is to formalize what is currently an ad-hoc process, to filter out ontologies that are clearly out of scope for OBO. It is intentionally weaker than full
+conformance, because (a) we want to allow registration of imperfect ontologies (b) partial conformance is easier to quickly and objectively check.
+
+Note this does not change the 2 week rule. Anyone from the OBO community is free to object to the ontology on the basis of any of these checks. 
+However, if no objections are raised in the 2 week period, then the ontology becomes registered. Once registered an ontology is never unregistered.
+
+We use ISO language here for SHOULD/MUST. Anyone is free to raise an objection based on a SHOULD but this is informative and non-blocking. 
+If a MUST is clearly violated then this is basis for exclusion/delaying registration.
+
+
+The checklist is intended for
+1. OBO Operations Committee members to evaluate the requests
+2. Ontology maintainers to ensure their ontology is eligible for admission prior to making a request
+
+The checklist is expected to evolve over time; anyone can contribute to the discussion [here](https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1116).
+
+## Checklist
+
+- Principle 1: [Open](http://obofoundry.org/principles/fp-001-open.html)
+  - The ontology MUST be conformant with this principle
+- Principle 2: [Common Format](http://obofoundry.org/principles/fp-002-format.html)
+  - The ontology MUST be conformant with this principle
+  - The ontology MUST be parseable in Protégé, ROBOT or OWLAPI and be logically consistent with a standard OWL reasoner such as ELK or HermiT
+- Principle 3: [URI/Identifier Space](http://obofoundry.org/principles/fp-003-uris.html) 
+  - The ontology MUST be conformant with this principle, with the caveat the prefix is not yet registered
+  - The requested namespace MUST be available (not already used by another ontology)
+- Principle 4: [Versioning](http://obofoundry.org/principles/fp-004-versioning.html) 
+  - The ontology MUST have a versionIRI
+- Principle 5: [Scope](http://obofoundry.org/principles/fp-005-delineated-content.html) 
+  - IF there is significant conceptual overlap or matching labels THEN there MUST be computable linkages (one of: skos, obo:xref, owl) which are reasonably comprehensive
```

Comment:
"IF there is significant conceptual overlap" --> "IF there is significant conceptual overlap to ontologies in OBO"